test/fail_compilation/fail15089.d

```diff
@@ -0,0 +1,10 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail15089.d(10): Error: cannot implicitly convert expression (130) of type int to byte
```

Comment:
If you put two expressions on the same line, and are confused about which one is the error, I'd edit the file and put them on separate lines.

After all the AST rewrites the compiler goes through, getting the original back can be problematic beyond just pointing to the source line.